README.md

```diff
@@ -1,26 +1,20 @@
-# Registry register data
+# Registers Registry
 
-The data for the
-[registry register](http://registry.openregister.org),
-[register register](http://register.openregister.org),
-[field register](http://field.openregister.org) and
-[datatype register](http://datatype.openregister.org) in each
-environment is currently maintained as individual yaml files in the
-[data](data/) directory.
+This repository contains the master copies of all registers currently managed
```

Comment:
can we say _hosted_ instead of _managed_? As I suppose you could say the Custodians manage the data and we host it.